jquery.xblock.js

```diff
@@ -161,22 +167,23 @@
 
             $('.xblock', root).not('.xblock-initialized').each(function(index, blockDOM) {
                 var initFnName = $(blockDOM).data('init');
-                if (initFnName) {
-                    // Don't fail when the page still contains XModules
-                    if (initFnName === 'XBlockToXModuleShim') {
-                        console.log('Warning: Unsupported XModule JS init', blockDOM);
-                        return;
-                    }
+                if (!initFnName) {
```

Comment:
@Kelketek this is not right. There're some XBlocs that doesn't have `initFn` but still work ok (at least one: OfficeMix). So, init should not short-circuit here. Please restore this code block to previous value (+your logging improvements)